### ISSUE TITLE
Is this project still alive 

### ISSUE THREAD

**Issue by elboukhari** [Filiosoft/npm-audit-html#93]
This repo seems to be inactive since many years.

Some of the dependencies are vulnerable :
```
https://snyk.io/advisor/npm-package/got
https://snyk.io/advisor/npm-package/marked
```

Any update please ?

**Comment by nprail** [comment]
Yeah, I haven't touched this project in awhile and don't really use it anymore either. I would be open to handing it off to someone who can maintain it.

**Comment by davityavryan** [comment]
Hey @nprail 
I can maintain it if it's ok for you.
We are kind of colleges, I would say.
I own [yarn-audit-html](https://www.npmjs.com/package/yarn-audit-html) and am planning to cover npm audit too.

**Comment by TaylorHo** [comment]
@elboukhari I have created an alternative version, based on this package.
It's **lightweight**, has **no vulnerable dependencies**, and **works offline** (without external resource requests).

Here are the links of [the repository](https://github.com/hotaydev/audit-export) and the [package on npm](https://www.npmjs.com/package/audit-export).

**Comment by TaylorHo** [comment]
Hey @nprail!

Hope you're doing well! 😊 I was thinking it might be a good idea to submit a pull request to the project. I want to add a friendly message to the README.md suggesting folks give [audit-export](https://www.npmjs.com/package/audit-export) a try, as it's actively maintained.

What do you think?

**Comment by rdepew** [comment]
Kudos to those who created **npm-audit-html**, and to those who picked it up and turned it into **audit-export**. NPM Audit makes lovely text output, but it needed something like this. I'm building **audit-export** into my Azure pipelines.

**Comment by carlosnat** [comment]
nice tools! the inspiration and the new thanks a lot
